#!/usr/bin/env python
PACKAGE = "progetto1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

int_enum = gen.enum([ gen.const('Euler', int_t, 0, "Euler Integration"),
                      gen.const('Runge_Kutta', int_t, 1, "Runge Kutta Integration")],
                    "An enum to set integration method")

gen.add("integration_method", int_t, 0, "0 for Euler, 1 for Runge Kutta", 0, 0, 1, edit_method = int_enum)

exit(gen.generate(PACKAGE, "scout_our_odom", "parameters"))